src/app/page.tsx

```diff
@@ -1,95 +1,27 @@
-import Image from "next/image";
-import styles from "./page.module.scss";
+"use client";
+import Styles from "./page.module.scss";
+import { SearchPanel } from "@/features/SearchPanel/SearchPanel";
+import { useEffect, useState } from "react";
+import { ArticleAPI } from "@/entities/Article";
+import { fetchNews } from "./api/fetchNews";
+import { Articles } from "@/features/Articles/Articles";
+import { schemaResponseAPI } from "./api/constants";
+import { z } from "zod";
 
 export default function Home() {
-  return (
-    <main className={styles.main}>
-      <div className={styles.description}>
-        <p>
-          Get started by editing&nbsp;
-          <code className={styles.code}>src/app/page.tsx</code>
-        </p>
-
-        <div>
-          <a
-            href="https://vercel.com?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            By{" "}
-            <Image
-              src="/vercel.svg"
-              alt="Vercel Logo"
-              className={styles.vercelLogo}
-              width={100}
-              height={24}
-              priority
-            />
-          </a>
-        </div>
-      </div>
-
-      <div className={styles.center}>
-        <Image
-          className={styles.logo}
-          src="/next.svg"
-          alt="Next.js Logo"
-          width={180}
-          height={37}
-          priority
-        />
-      </div>
+  const [articles, setArticles] = useState<ArticleAPI[]>([]);
+  const [error, setError] = useState("");
+  useEffect(() => {
+    fetchNews()
+      .then((data) => setArticles(data.articles))
```

Comment:
Replace it with async await for consistency

Reply:
But what is the benefit of doing it? Doesn't it look easier this way?